core/src/main/java/org/apache/calcite/adapter/enumerable/StrictAggImplementor.java

```diff
@@ -148,13 +146,7 @@ public final void implementAdd(AggContext info, final AggAddContext add) {
       return;
     }
 
-    final Map<RexNode, Boolean> nullables = new HashMap<>();
-    for (RexNode arg : args) {
-      if (translator.isNullable(arg)) {
-        nullables.put(arg, false);
-      }
```

Comment:
Can you explain why we put `false` here for nullable `true` ?

Reply:
Because the initial algorithm assumes the null checking logic for `args` has already been conducted, and it wants to tell the `translator` that "you can treat these RexNode as nullable=false".